Tasks/AzureResourceGroupDeploymentV2/task.json

```diff
@@ -14,8 +14,8 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 2,
-        "Minor": 175,
-        "Patch": 2
+        "Minor": 178,
```

Comment:
Please ensure that this version is still greater than latest as the PR is a bit old now.

Reply:
I have set both to the current sprint number of 179.